examples/hd44780/customchar/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"machine"
+
+	"github.com/tinygo-org/drivers/hd44780"
+)
+
+func main() {
+
+	lcd, _ := hd44780.NewGPIO4Bit([]uint8{machine.P0, machine.P1, machine.P2, machine.P3}, machine.P4, machine.P5, machine.P6)
```

Comment:
Maybe put each param in a new line to make it more clear to read, and also for the configuration below (same for the other example)